geode-junit/build.gradle

```diff
@@ -19,6 +19,10 @@ apply from: "${rootDir}/${scriptDir}/standard-subproject-configuration.gradle"
 
 apply from: "${project.projectDir}/../gradle/publish-java.gradle"
 
+compileJava {
+  options.compilerArgs << '-Xlint:-sunapi'
+  options.compilerArgs << '-XDenableSunApiLintControl'
```

Comment:
So we've moved from Bouncy Castle, a high quality open source project, to an internal JDK class. These compiler options suppress what would be compile errors I suppose (or at least warnings) that we are illegally accessing JDK internal classes.

My research shows that in JDK 9 these classes in a `sun` package will move to a `javax` one. I suppose when that happens the JDK8 internal classes accessed here will have direct replacements available so all we'll have to do is change the includes in these files.